### PR TITLE
Feature/proxy attribute forwarding

### PR DESCRIPTION
**Pull request by sebclrsn**
This PR implements the cchannel API forwarding mechanism discussed as partial solution 2 for complex connectors.

It also fixes the activate_log functionality so that when loggers are activated by one auxiliary, they aren't deactivated by the following ones

### CODE REVIEW

File: docs/whats_new/version_ongoing.rst [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+
+Internal creation of proxy auxiliaries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is no longer necessary to manually defined a ``ProxyAuxiliary`` with
+``CCProxy``s yourself. If you simply pass the communication channel to
+each auxiliary that has to share it, ``pykiso`` will do the rest for you.
+
+
+Remove reduncancy of activate_log configuration parameter
```
Comment (Pog3k) [inline docs/whats_new/version_ongoing.rst:12]:
```suggestion
Remove redundancy of activate_log configuration parameter
```

File: docs/advanced_usage/advanced_config_file.rst [hunk 2]
```diff
@@ -131,6 +131,12 @@ that will have exclusive access to the communication channel. A
 the created :py:class:`~pykiso.lib.auxiliaries.proxy_auxiliary.ProxyAuxiliary`. This allows to
 keep a minimalistic :py:class:`~pykiso.connector.CChannel` implementation.
 
+Access to attributes and methods of the defined communication channel that has been
+attached to the created :py:class:`~pykiso.lib.auxiliaries.proxy_auxiliary.ProxyAuxiliary`
+is still possible, but keep in mind that if one auxiliary modifies one the communication
+channel's attributes, every other auxiliary sharing this communication channel will be
+affected by this change.
```
Comment (Pog3k) [inline docs/advanced_usage/advanced_config_file.rst:138]: I think we should illustrate that with a python snippet somewhere

**Review by Pog3k — APPROVED** [review]

File: docs/whats_new/version_ongoing.rst (hunk 1 above)
Comment (sebclrsn) [reply docs/whats_new/version_ongoing.rst:12]: To me it is sufficient to have this behaviour explained with text and then illustrated with the schema linked below, adding a code snippet here will add a lot of content for a side-note the user shouldn't usually have to care about

File: docs/advanced_usage/advanced_config_file.rst (hunk 2 above)
Comment (sebclrsn) [reply docs/advanced_usage/advanced_config_file.rst:138]: To me it is sufficient to have this behaviour explained with text and then illustrated with the schema linked below, adding a code snippet here will add a lot of content for a side-note the user shouldn't usually have to care about

File: docs/whats_new/version_ongoing.rst [hunk 3]
```diff
@@ -1,2 +1,19 @@
...
+
+Internal creation of proxy auxiliaries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is no longer necessary to manually defined a ``ProxyAuxiliary`` with
+``CCProxy``s yourself. If you simply pass the communication channel to
+each auxiliary that has to share it, ``pykiso`` will do the rest for you.
+
+
+Remove redundancy of activate_log configuration parameter
```
Comment (sebastianpfischer) [inline docs/whats_new/version_ongoing.rst:12]: Where is that coming from?

File: src/pykiso/auxiliary.py [hunk 4]
```diff
@@ -26,9 +26,9 @@
 import time
 from typing import Any
 
-from .logging_initializer import add_logging_level
-from .test_setup.config_registry import ConfigRegistry
-from .types import MsgType
+from pykiso.logging_initializer import add_internal_log_levels
```
Comment (sebastianpfischer) [inline src/pykiso/auxiliary.py:29]: Please cluster the features in the following way: one feature <-> one PR. Else, there is a miss-match between your PR title, why it exists and its content.

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]

**Comment by sebastianpfischer** [comment]
Please separate the PR in 2 with clear descriptions for each and the feature you want to add. Thanks!

File: src/pykiso/auxiliary.py (hunk 4 above)
Comment (sebclrsn) [reply src/pykiso/auxiliary.py:29]: That's not a feature but an extremely small refactoring at auxiliary level that just uses an API from a previous PR

I'll make a separate PR for the activate_log feature modification, but this one can definitely stay in here IMO

File: docs/whats_new/version_ongoing.rst (hunk 3 above)
Comment (sebclrsn) [reply docs/whats_new/version_ongoing.rst]: Now part of another PR

**Review by BKaDamien — APPROVED** [review]

**Pull request merged**